google/cloud/spanner/benchmarks/multiple_rows_cpu_benchmark.cc

```diff
@@ -797,6 +789,7 @@ class ReadExperiment : public Experiment {
  *     `config.query_size` rows
  *   - Measure the CPU time required by the previous step
  */
+
```

Comment:
nit: we normally do not put blank lines here (there are other new blank lines too?).